### PR TITLE
Use latest version of heroics

### PR DESCRIPTION
**Pull request by rramsden**

### CODE REVIEW

**Comment by rramsden** [comment]
@camelmasa Can you test this?

**Comment by camelmasa** [comment]
I'll review it 😃

File: lib/komoju/client.rb [hunk 1]
```diff
@@ -163,10 +170,8 @@ def initialize(client)
     end
 
     # List events
-    #
-    # @param collection_options: additional collection options to pass with the request
-    def list(collection_options = {})
```
Comment (camelmasa) [inline lib/komoju/client.rb:168]: I think we should not remove this option 😥 Looks we need to hack heroics ... 😓

Ref: previous option here https://github.com/komoju/komoju-ruby/commit/f67e4a4ff4a8009f652c203c8619afb7a9bf80af#diff-971db54e7f0e2821723c84cb193cdde7L168

File: lib/komoju/client.rb [hunk 2]
```diff
@@ -185,10 +190,9 @@ def initialize(client)
 
     # List payments
     #
-    # @param collection_options: additional collection options to pass with the request
     # @param body: the object to pass as the request payload
-    def list(collection_options = {}, body)
-      @client.payments.list(collection_options, body)
+    def list(body = {})
```
Comment (camelmasa) [inline lib/komoju/client.rb:194]: No problem. This method overrides above method ... little bit strange 😭

**Comment by camelmasa** [comment]
@rramsden LGTM 👍  It works for me.

**Comment by camelmasa** [comment]
@rramsden I merged this. Could you push to rubygems ❓

**Pull request merged**